src/azul/openapi/spec.py

```diff
@@ -232,3 +232,14 @@ def version(self) -> JSON:
                 }
             }
         }
+
+    @classmethod
+    def http_504_response(cls) -> JSON:
+        return {
+            '504': {
+                'description': 'Request timed out. When handling this response,'
+                               ' clients should wait the number of seconds'
```

Comment:
I don't think we typically place the space at the beginning of the continuation. If you disagree, please provide evidence.